tests/test_cmdLineFwk.py

```diff
@@ -590,15 +590,12 @@ def test_simple_qgraph_qbb(self):
             fwk.runGraphQBB(taskFactory, args)
 
             # Transfer the datasets to the butler.
-            # TODO: DM-40392
-            # This will fail because the UUIDs are not updated by updateRun
-            with self.assertRaises(ConflictingDefinitionError):
-                n2 = transfer_from_graph(temp_graph.name, self.root, True, False, False)
-
-                # For reasons that have to be investigated the number of
-                # outputs the second time around is only 21.
-                self.assertEqual(n2, 31)
-                self.assertEqual(n1, n2)
+            n2 = transfer_from_graph(temp_graph.name, self.root, True, False, False)
+
+            # For reasons that have to be investigated the number of
+            # outputs the second time around is only 21.
```

Comment:
Given that the test passes for you I think this means that the 21 I was getting was caused by the bad graph problems and you can remove this comment. We can likely remove the first test and only test that `n1 == n2`.